site/app/templates/submission/homework/LateDayMessage.twig

```diff
@@ -1,28 +1,13 @@
 {% import _self as self %}
 
-{% if messages|length > 0 or daylight %}
+{% if messages|length > 0 %}
 {# WRAP THE LATE DAY INFORMATION IN A DIV #}
-<div class="content"
+<div class="content gradeable_message"
+    {% if error %}
+        style="background-color: var(--standard-light-red);"
```

Comment:
It would be better to apply a class instead of inline CSS.

Reply:
Which css file would I write this to?

Reply:
I couldn't immediately find an external stylesheet relating to this specifically.  You could create a new stylesheet or add this to a more general stylesheet  In the interest of getting this merged ASAP, I think it's acceptable as-is.